### DISCUSSION TITLE
Executing test 1 spec pass and 1 scenario fail

### DISCUSSION THREAD

**Discussion by ramesh-khatri** [getgauge/taiko#1814] [General]
I am running a 1 spec which has only 1 scenario, after the test is executed it shows 1 spec pass and 1 scenario fail. 
<img width="368" alt="gauge_github (3)" src="https://user-images.githubusercontent.com/76654941/105316687-52d30200-5be9-11eb-998f-cbe983c01c39.png">

I realize that this happens as my scenario is pass but after spec failed. I have not defined afterSpec . Is there any default set in afterSpec?

**Comment by NivedhaSenthil** [comment]
Please share a sample for us to replicate this issue

**Reply by ramesh-khatri** [reply]
Actually, I am not sure either this is the issue or not. I tried to replicate this issue on a simple task by adding some assertions but I was unable to replicate it. But, in my below code where I have used the different library to read, write the pdf and I get that error
```
step("This is the test step name", async () => {
    var downloadPath = path.resolve(__dirname, '../../data/file/downloaded');
    await client().send('Page.setDownloadBehavior', {
        behavior: 'allow',
        downloadPath: downloadPath,
    });
    await click(link('Download'))
    await waitFor(() => fs.existsSync('data/file/downloaded/resume.pdf'))
    pdfParser.loadPDF('data/file/downloaded/resume.pdf');
    pdfParser.on("pdfParser_dataError", errData => console.error(errData.parserError));
    pdfParser.on("pdfParser_dataReady", async() => {
        let pdfContent = pdfParser.getRawTextContent()
        fs.writeFileSync("data/file/downloaded/resume.txt", pdfContent)
        var data = fs.readFileSync('data/file/downloaded/resume.txt','utf8');
        expect(data).to.have.string('John Doe')  // This is valid assertion
        expect(data).to.have.string('john.doe@gmail.comtdtgss')    //This is invalid assertion
    });     
});
```
Also, the condition 1 spec fail and 1 scenario pass doesn't occur every time. Sometimes it shows both fail also.
